src/js/utils/option-utils.jsx

```diff
@@ -171,3 +171,20 @@ export const debounceLocationGroupsFetch = (waitTime, minSearchLength) =>
       callback([]);
     }
   }, waitTime);
+
+export const organizationsFetch = (roleTypes = ['ROLE_SUPPLIER'], active = false) =>
```

Comment:
Add action fetchSuppliers, and store it in the reducer. Why it has `active = false` by default?

Reply:
I believe when adding active column to party, we didn't want to create much regression and to set `active=true`, because that would make organizations not visible in many places where they should be visible. This conception is followed everywhere where fetching parties.